cloudbuild-e2e-gae.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  # Wait for the image to exist
+  - name: "docker"
+    id: wait-for-image
+    entrypoint: "sh"
+    timeout: 10m
+    env: ["_TEST_SERVER_IMAGE=${_TEST_SERVER_IMAGE}"]
+    args:
+      - e2e-test-server/wait-for-image.sh
+
+  # Run the test
+  - name: $_TEST_RUNNER_IMAGE
+    id: run-tests-gae
+    dir: /
+    env: ["PROJECT_ID=$PROJECT_ID"]
+    args:
+      - gae
+      - --image=$_TEST_SERVER_IMAGE
+      - --runtime=node
+
+logsBucket: gs://opentelemetry-ops-e2e-cloud-build-logs
+timeout: 20m
+substitutions:
+  _TEST_RUNNER_IMAGE: gcr.io/${PROJECT_ID}/opentelemetry-operations-e2e-testing:0.15.0
```

Comment:
can/should you test this against 0.16.0?

Reply:
I'd have to update the exporter config for tests to pass through those service. I can do it in another PR